plugins/inputs/ipmi_sensor/ipmi.go

```diff
@@ -119,6 +131,29 @@ func (m *Ipmi) parse(acc telegraf.Accumulator, server string) error {
 		opts = conn.options()
 	}
 	opts = append(opts, "sdr")
+	if m.UseCache {
+		cacheFile := filepath.Join(m.CachePath, server+"_ipmi_cache")
```

Comment:
You should check if `CachePath` provided by user exists and is writeable directory.

Reply:
CachePath defaults to systems tmp dir which exists and is writable by default. Don't you think checking the directory is a bit out of scope of the application and should be left to the user when changing config?

Reply:
Honestly, I think either it should be checked here, or in README (and sample config) it should be stated that user needs to provide path to writeable directory where cache file can be stored. 
For security reasons, I prefer the former.

Reply:
In my eyes thats just a logical consequence for a given tmp dir (cache dir) that is must exist and must be writable. Anyway I added that to the description and readme now.
Since use cache is now off by default I don't see the need for adding more code for bizarre cases.